controlplane/src/core/bufservices/subgraph/publishFederatedSubgraph.ts

```diff
@@ -189,7 +189,9 @@ export function publishFederatedSubgraph(
             details:
               subgraph.type === 'grpc_plugin'
                 ? `Subgraph ${subgraph.name} is a plugin. Please use the 'wgc router plugin publish' command to publish the plugin.`
-                : `Subgraph ${subgraph.name} is not of type ${formatSubgraphType(req.type)}.`,
+                : subgraph.type === 'grpc_service'
+                  ? `Subgraph ${subgraph.name} is a grpc service. Please use the 'wgc grpc-service publish' command to publish the grpc service.`
+                  : `Subgraph ${subgraph.name} is not of type ${formatSubgraphType(req.type)}.`,
```

Comment:
I don't like where this ternary is going, can we clean this up to be flat for any number of subgraph types?